src/components/fields/AnyOfField.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { guessType } from "../../utils";
+
+class AnyOfField extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedOption: 0,
```

Comment:
It just occurred to me that this might not be safe if you are rendering a form from existing data. Should this try to guess the index based on the formData/anyOf schemas?

Reply:
Yeah I think thats sensible behaviour, I'll add some tests for it as well